templates/webapps/galaxy/tool_form.mako

```diff
@@ -3,455 +3,22 @@
 ${h.js("libs/bibtex", "libs/jquery/jquery-ui")}
 ${h.css('base', 'jquery-ui/smoothness/jquery-ui')}
 
-## skip new tool form code if disabled
-%if util.string_as_bool(trans.app.config.get('toolform_upgrade',  True)):
```

Comment:
Ahh, got it -- I hadn't put it together that John was talking about this when he said 'end the old tool form', somehow.  Super short deprecation warning, we *probably* need to handle this and not just purge it all quite yet?